src/pages/en/guides/cms/netlify-cms.md

```diff
@@ -13,3 +13,11 @@ service: Netlify CMS
 
 - You can get started with the [Astro Blog Starter with Netlify CMS](https://github.com/delucis/astro-netlify-cms-starter).
 
+- Blog Post: [Author your Astro site's content with Git-based CMSs](https://aalam.vercel.app/blog/astro-and-git-cms-netlify) by Aftab Alam 
+
+## Production Sites
+
+The following sites use Astro + Netlify CMS in production:
+
+- [Public Transport Forum New Zealand](https://publictransportforum.nz/articles) 
+
```

Comment:
I can’t seem to find the source repo for this site. I’m not against including production examples, but without source code to check _how_ something was done, I’m not sure they offer that much value.

```suggestion
```

Reply:
Some open source sites we could link to:

- Duluth MakerSpace — https://duluth-makerspace.github.io/ — https://github.com/Duluth-MakerSpace/duluth-makerspace.github.io
- Yuniel Acosta — https://www.yunielacosta.com/ — https://github.com/yacosta738/yacosta738.github.io

Reply:
> Hmm... this is @zadeviggers -- ZV, I don't see the source code public on your GH site. Is it available somewhere?

Unfortunately, it's not open-source sorry - it's a client project (that I continue to hack on for fun), and I can't share the source.

If you decide to keep it in the list (of course, I completely understand if you remove it), the site now also heavily uses `@astrojs/image` to optimise images and normalise the formats they're served in, which is the secret to its legendary score on the hyperdrive.